src/cplscheme/config/CouplingSchemeConfiguration.cpp

```diff
@@ -531,6 +535,7 @@ void CouplingSchemeConfiguration::addTagAbsoluteConvergenceMeasure(
   XMLTag tagConvergenceMeasure(*this, TAG_ABS_CONV_MEASURE, XMLTag::OCCUR_ARBITRARY);
   addBaseAttributesTagConvergenceMeasure(tagConvergenceMeasure);
   XMLAttribute<double> attrLimit(ATTR_LIMIT);
+  attrLimit.setDocumentation("Limit under which the measure is considered to have converged.");
```

Comment:
Perhaps a better description of each convergence measure type? From the absoluteConvergenceMeasure.hpp

 " The convergence is evaluated by looking at the two norm of the differences between each data value from the new and old data set. If the two norm is equal or below a given limit, convergence is achieved."

We can add this for the others below?